src/datreant/core/trees.py

```diff
@@ -175,7 +179,7 @@ def __init__(self, dirpath, limbs=None):
             self.attach(*limbs)
 
     def __repr__(self):
-        return "<Tree: '{}'>".format(self.relpath)
+        return "<Tree: '{}'>".format(self.abspath)
```

Comment:
? oh...that should be relpath.

relpath on win can't work if two harddisks are used within the path. That why i used a seperate tmpdir for the test.